scripts/common-run.sh

```diff
@@ -54,6 +54,13 @@ rsyslog_log_format() {
 	sed -i -E "s/<log-format>/${log_format}/" /etc/rsyslog.conf
 }
 
+logrotate_remove_duplicate_mail_log() {
+	if egrep -q '^/var/log/mail.log' /etc/logrotate.d/logrotate.conf; then
```

Comment:
There's a knob to configure `/etc/logrotate.d/logrotate.conf` so I'm only removing it from `/var/log/mail.log` at runtime as a function of what the actual config is.